webroot/results/js/data_upload_help.js

```diff
@@ -62,4 +62,8 @@ $(document).ready(function() {
   selectize_competition_field("#form-element-competitionId");
   get_comp_info(0);
   $('#form-element-competitionId').change(get_comp_info);
+  // Prevent from multiple form submissions.
```

Comment:
nit: You can drop the "from", simply "Prevent multiple form submissions" will do!